docker-compose.yml

```diff
@@ -168,6 +168,21 @@ services:
             networks:
                 net:
                     ipv4_address: 10.11.3.0
+        electrs:
+              container_name: electrs
+              image: getumbrel/electrs:v0.8.5
+              logging: *default-logging
+              depends_on: [ bitcoin ]
+              volumes:
+                - ${PWD}/bitcoin:/data/.bitcoin:ro
+                - ${PWD}/electrs:/data
+              restart: unless-stopped
+              stop_grace_period: 1m
```

Comment:
I've not tried it out, but do you think `1m` is enough or shall we increase this to `5m` or so?

Reply:
I've not noticed it being force killed but yeah no harm in increasing it a little to be safe.